webapp/src/components/App/Singlestudy/HomeView/InformationView/CreateVariantDialog.tsx

```diff
@@ -58,7 +58,7 @@ function CreateVariantDialog(props: Props) {
 
   return (
     <FormDialog
-      maxWidth="sm"
+      maxWidth="xs"
```

Comment:
please revert that change, studies names can be quite long so `xs` width is too small in most cases. And their is no width constraints when opening the dialog

Reply:
ok